templates/gmond.conf.el5.erb

```diff
@@ -95,31 +137,40 @@ collection_group {
   time_threshold = 1200 
   metric { 
     name = "cpu_num" 
+    title = "CPU Count"
   } 
   metric { 
     name = "cpu_speed" 
+    title = "CPU Speed"
   } 
   metric { 
     name = "mem_total" 
+    title = "Memory Total"
   } 
   /* Should this be here? Swap can be added/removed between reboots. */ 
   metric { 
     name = "swap_total" 
+    title = "Swap Space Total"
   } 
   metric { 
     name = "boottime" 
+    title = "Last Boot Time"
   } 
   metric { 
     name = "machine_type" 
+    title = "Machine Type"
   } 
   metric { 
     name = "os_name" 
+    title = "Operating System"
   } 
   metric { 
     name = "os_release" 
+    title = "Operating System Release"
   } 
   metric { 
-    name = "location" 
+    name = "location"
```

Comment:
It might just be github being glitchy when refs are rebased.  I'm going to close this PR since the requester appears to have abandoned it.